backend/middleware/sanitize.ts

```
@@ -337,7 +337,7 @@ export function sanitize(value, options = {}) {
   let result;
   if (typeof value === 'string') {
     // Normalize Unicode and remove zero-width chars
-    let clean = value.normalize('NFC').replace(/[\u200B-\u200D\uFEFF]/g, '');
+    const clean = value.normalize('NFC').replace(/[\u200B-\u200D\uFEFF]/g, '');
```

Biome (1.9.4)

[error] A character class cannot match a joined character sequence.

A zero width joiner composes several emojis into a new one. Replace the character class with an alternation.

(lint/suspicious/noMisleadingCharacterClass)